x-pack/plugins/uptime/public/components/functional/monitor_status_bar.tsx

```diff
@@ -50,19 +50,21 @@ export const MonitorStatusBar = ({ timestamp, url, duration, status }: Props) =>
           </EuiLink>
         </EuiFlexItem>
       </EuiFlexItem>
-      <EuiFlexItem
-        aria-label={i18n.translate('xpack.uptime.monitorStatusBar.durationTextAriaLabel', {
-          defaultMessage: 'Monitor duration in milliseconds',
-        })}
-        grow={false}
-      >
-        <FormattedMessage
-          id="xpack.uptime.monitorStatusBar.healthStatus.durationInMillisecondsMessage"
-          values={{ duration: duration ? duration : 0 }}
-          defaultMessage="{duration}ms"
-          description="The 'ms' is an abbreviation for 'milliseconds'."
-        />
-      </EuiFlexItem>
+      {!!duration && (
```

Comment:
I think we're ok to just omit this element if there's no `duration` for some reason. There should always be one though.

Reply:
++